documentation/docs/index.md

```diff
@@ -17,6 +17,11 @@
 ## Installation
 
 ### Download the repo with binary assets
+!!! tip "Choosing a location to download the repo"
+
+      Before running the commands, make sure you’re in the correct folder (the "current working directory") where you want to save the repo locally. You can use the `pwd` command to print the current directory, and `cd` to change the directory. You should ensure that there are no spaces at any point in the current working directory path. For example, if your top level user folder contains a space in the name, consider downloading the repo to a folder located outside of the user folder, e.g. in a new top-level `C:/apps` folder on Windows.
+
```

Comment:
nit(blocking)
I think we can trim this down a bit, how about:

> It is best to download the repo to a folder path that does not include any spaces. E.g. if using a user documents directory that includes spaces `/user/my name` you may need to instead re-open the terminal in a different folder instead.

I might also suggest that we add a troubleshooting section where we put this instead along with likely error message/identifier, as I expect this won't really be an issue for the majority of users.

Reply:
I've appended it to be your shorter version, much better.

I know what you mean about this not being relevant to most users. And generally I don't like having parts of the main docs that go into detail about specific potential problems. But in this case I think having it in an isolated box upfront might be worth it – an error was only seen at the point of running `yarn workflow sync` after completing all the previous setup steps, at which point a library threw an error along the lines of `No such file or directory: '/user/my'` (splitting the path at the space). I think documenting in this way could catch a few cases without causing much distraction for most users. 

Although possibly I'm motivated by an awareness that there's quite a long list of possible issues that could go in a "troubleshooting" section of the docs...

Reply:
Sounds good, you're right that it's a confusing error to lookup in a troubleshooting guide (would still be good to add one in the future). Let's keep the main info box for now then and I'll create a follow-up low-priority issue https://github.com/IDEMSInternational/open-app-builder/issues/2411